dali/operators/crop/bbox_crop.h

```diff
@@ -122,7 +122,7 @@ class RandomBBoxCrop : public Operator<Backend> {
     return sampler(rngs_[sample]);
   }
 
-  bool ValidAspectRatio(float width, float height) const {
+  bool ValidAspectRatio(float height, float width) const {
```

Comment:
you changed the behavior here. Was it wrong before?

Reply:
1) Other functions follow that order (height then width) - like RemapBoxes
2) `ValidAspectRatio` is invoked with height as the first argument then the width

Reply:
yes, but you didn't change the body of the function so you are getting the inverse ratio compared to what you used to get before this change. I am not sure which one is the correct though

Reply:
In this particular function width is still width so no changes are required here. In case of caller, arguments have been swapped and it was a bug.

Reply:
Also in SSD aspect ratio is in range 0.5-2.0 so it is symmetric.

Reply:
In general, the aspect ratio is given as width : height. Think screen: 16:9, 4:3, etc. It would probably be better to keep the arguments as (width, height) and switch them at call site.
Anyway, that's a problem that keeps plaguing other APIs, too - like numpy, opencv - whenever you move from thinking about arrays to thinking about images, the order is switched. I guess it just can't be avoided in a library that mixes arrays/tensors with graphics.

Reply:
Agree. Still, in that case, I wanted to have a consistent API with other functions that accept height and then width.
In the next PR, we can change all functions in this operator to accept width and then height.